time stamp
float64 totalTime
int32 nq
int32 bezierOrder
float64[] controlPoints